app/assets/javascripts/oxalis/controller/combinations/skeletontracing_plane_controller.js

```diff
@@ -215,6 +215,8 @@ class SkeletonTracingPlaneController extends PlaneControllerClass {
     if (centered) {
       // we created a new node, so get a new reference
       getActiveNode(Store.getState().tracing).map(newActiveNode =>
+        // Center the position of the active node without modifying the "third" dimension (see centerPositionAnimated)
```

Comment:
I think, this comment / the code could be a bit clearer. The point of the comment is that `centerPositionAnimated` must not receive `false` as a second parameter, right? Since the `skipDimensions` parameter is optional and defaults to (in this case desired value of) true, it's hard to understand this comment. Maybe just pass the `true` parameter explicitly?